oshi-core/src/main/java/oshi/driver/windows/wmi/Win32Process.java

```diff
@@ -89,7 +89,7 @@ public static WmiResult<ProcessXPProperty> queryProcesses(Collection<Integer> pi
         String sb = WIN32_PROCESS;
         if (pids != null) {
             sb += " WHERE ProcessID="
-                    + pids.stream().map(String::valueOf).collect(Collectors.joining(" OR PROCESSID="));
+                    + pids.stream().parallel().map(String::valueOf).collect(Collectors.joining(" OR PROCESSID="));
```

Comment:
Same as above. I'm not sure whether a random sort order of the process ID filters would make any difference on query speed.  Probably not needed.